.azure-pipelines.yml

```diff
@@ -21,6 +21,7 @@ variables:
   GO_DEP_VERSION: 0.5.4
   PYTHON_PIPENV_VERSION: 2018.11.26
   RUST_VERSION: 1.35.0
+  SCANCODE_VERSION: 3.2.1rc2
```

Comment:
Should this be squashed to the previous commit, so really each commit in this PR would pass tests on its own?

Reply:
Also, is this change really necessary as ScanCode still would get bootstrapped, or?

Reply:
> 
> 
> Should this be squashed to the previous commit, so really each commit in this PR would pass tests on its own?

I have instead merged it with a new commit that removes the committed ScanCode license files which I forgot to do previously. FYI, I have also rewritten the commit that introduces the function to locate the ScanCode license dir to automatically do this in the license text functions. Otherwise all callers of those functions would have had to do this. This was actually the reason why this was put on hold until the utils refactoring was done.

Reply:
> 
> 
> Also, is this change really necessary as ScanCode still would get bootstrapped, or?

Only by the `ScanCodeScannerFunTest`, but this would not help for the license text related tests which do not call the `ScanCode` binary at all.

Reply:
It seems like detecting the ScanCode dir does not work on CI, I'll have to look into this next week.

Reply:
Commit message nit: the "previous commit" reference does not seem right anymore.